app/models/solr_document.rb

```diff
@@ -72,6 +72,7 @@ def linked_publications(params)
                                                    search_builder_class: PublicationSearchBuilder)
     response, _deprecated_stuff = search_service.search_results
     docs = response.documents
```

Comment:
Should we cache this so it doesn't make 2 of these queries for each page draw?

Reply:
Probably not necessary for the anticipated 1 simultaneous user. But I defer to your judgement and recommendation on how/where to cache.

Reply:
How about something like this?
```ruby
  return @publications unless @publications.nil?
  response, _deprecated_stuff = search_service.search_results
    docs = response.documents
    return @publications = false if docs.empty?
    tuples = docs.map do |doc|
      "<li><a href=\"#{search_link(doc.id)}\">#{doc.title}</a> (#{doc.first('created_year_isim')})</li>"
    end
    @publications = "<ul>#{tuples.join}</ul>".html_safe
end

Reply:
Works for me. Fixing.

Reply:
Required a little bit of tweaking because returning false wasn't equivalent to returning nil.